public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { locationService } from '@grafana/runtime/src/services/LocationService';
+import { useStyles2 } from '@grafana/ui';
+import { AlertLocalStorage } from 'app/percona/shared/components/Elements/AlertLocalStorage/AlertLocalStorage';
+import { getPerconaSettings } from 'app/percona/shared/core/selectors';
+import { useSelector } from 'app/types';
+
+import { Messages } from './CheckInfoAlert.messages';
+import { getStyles } from './CheckInfoAlert.styles';
+
+export const ChecksInfoAlert = () => {
+  const { result } = useSelector(getPerconaSettings);
+  const { isConnectedToPortal } = result!;
+  const styles = useStyles2(getStyles);
+
+  return (
+    <>
+      {!isConnectedToPortal && (
```

Comment:
You could do an early return here.

```ts
if (isConnectedToPortal) {
   return null;
}

return (<AlertLocalStorage ... />);
```

This way you would avoid the fragment and coditional in component render.

Reply:
Good tip :)